packages/codemods/src/utils/styled-components-attributes.ts

```diff
@@ -37,7 +37,7 @@ const getRestAttributeExpressions = (expression: ObjectExpression) =>
     ? (expression.properties.filter((property) => !isStyleProperty(property)) as ObjectProperty[])
     : null;
 
-const getAlternativeComponentName = (name: string) => `Compiled${name}`;
+const getAlternativeComponentName = (name: string) => `${name}Styles`;
```

Comment:
We had a hard time coming up with a description name for this component. It's the component that's had any styles that came from `.attr` applied and awaiting the rest of props from `.attr` to be spread back in. Any thoughts on the naming of this?